index.html

```diff
@@ -1,52 +1,33 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
https://www.w3.org/TR/html4/

tl;dr "hey guys this spec is 20 years old please don't use it"

Reply:
20 years?
TWENTY?

my god, I'm old.